e2e/flows/authentication.flow.ts

```diff
@@ -31,6 +31,50 @@ export class AuthenticationFlow {
         await this.waitForAngularIfRequired();
     }
 
+    async signInAsHomeOfficeApc() {
+        await this.signOut();
+        await this.idamSignInPage.waitUntilLoaded();
```

Comment:
I am thinking we could do refactoring similar to the above here as well.

Reply:
As per the existing code base. Could consider rebasing in the future.